less/flipcard.less

```diff
@@ -30,21 +30,27 @@
 
 // flipcard component styling
 .flipcard-component {
+  
+   // IE fix for jumping Footer nav
+   .flipcard-inner {
+     overflow: hidden;
+   }
+  
   .flipcard-multiple {
     width: 50%;
 
-    .flipcard-item-face {
-      width: 99%;
-    }
-  }
-
-  .flipcard-single {
-    width: 100%;
-
     .flipcard-item-face {
       width: 100%;
     }
   }
+  
+   .flipcard-single {
+     width: 100%;
+
+      flipcard-item-face {
```

Comment:
Can you double check that this is actually correct.
Looks like the `.` for the class is missing here.